src/testIntegration/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/security/SmoketestTest.kt

```diff
@@ -0,0 +1,104 @@
+package uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.security
+
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.given
+import org.springframework.http.MediaType
+import org.springframework.web.reactive.function.BodyInserters
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.barcode.Barcode
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.barcode.BarcodeEvent
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.barcode.BarcodeEventType
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.barcode.BarcodeResourceTest
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.magiclink.VerifyLinkResponse
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.mocks.HmppsAuthExtension
+
+class SmoketestTest : BarcodeResourceTest() {
+
+  @Nested
+  inner class SmokeTestAllowed {
+
+    @BeforeEach
+    fun `smoke test configuration`() {
+      given(smokeTestConfig.lsjSecret).willReturn("some-lsj-secret")
+      given(smokeTestConfig.msjSecret).willReturn("some-msj-secret")
+    }
+
+    @Test
+    fun `A legal sender smoke test can sign in using the back door`() {
```

Comment:
Given that this is open source on github, do we want to use terms like 'back door'? Would something like `A legal sender smoke test can sign in as the smoke test user` be any better ?